views/partials/header.ejs

```diff
@@ -20,31 +20,25 @@
         </li>
         <% if (user.role === 'candidate' && !user.candidateProfile) { %>
         <li class="nav-item">
-          <a class="nav-link" href="<%=`/user/${user._id}/create_candidate` %>"> New Candidate </a>
+          <a class="nav-link" href="<%=`/user/${user._id}/create_candidate` %>"> New candidate </a>
         </li>
         <% } else if (user.role === 'recruiter') { %>
-        <li class="nav-item">
-          <a class="nav-link" href="<%=`/user/${user._id}/create_offers` %>"> Add job </a>
-        </li>
-        <li class="nav-item">
-          <a class="nav-link" href="<%=`/user/${user._id}/offers` %>">Your Jobs </a>
-        </li>
         <% } %>
         <li class="nav-item">
           <a class="nav-link" href="<%=`/user/${user._id}` %>"> Profile</a>
         </li>
         <li class="nav-item">
-          <a class="nav-link btn btn-primary my-1 my-xl-0" href="/user/logout"> Log Out </a>
+          <a class="nav-link btn btn-outline-primary my-1 my-xl-0" href="/user/logout"> Log out </a>
```

Comment:
![grafik](https://user-images.githubusercontent.com/38247392/103534743-c35eec80-4e8f-11eb-9fa4-4777b1697897.png)
This button needs more padding, for example you can add the class "px-3"